packages/Search/CMakeLists.txt

```diff
@@ -1,6 +1,18 @@
-ADD_SUBDIRECTORY(src)
+cmake_minimum_required(VERSION 3.13)
+project(ArborX CXX)
 
-TRIBITS_INCLUDE_DIRECTORIES(examples/point_clouds)
+find_package(kokkos REQUIRED)
+find_package(Boost REQUIRED COMPONENTS program_options unit_test_framework)
```

Comment:
Add `geometry`.

Reply:
You only list boost libraries that were built, not header-only packages.